metrics.go

```diff
@@ -77,12 +77,24 @@ var (
 		Buckets: durationPerBlockHistogram,
 	})
 
+	fetchDurationCarSuccessMetric = prometheus.NewHistogram(prometheus.HistogramOpts{
+		Name:    prometheus.BuildFQName("ipfs", "caboose", "fetch_duration_car_success"),
+		Help:    "Latency observed during successful caboose fetches for a car",
+		Buckets: durationPerBlockHistogram,
+	})
+
 	fetchDurationBlockFailureMetric = prometheus.NewHistogram(prometheus.HistogramOpts{
 		Name:    prometheus.BuildFQName("ipfs", "caboose", "fetch_duration_block_failure"),
 		Help:    "Latency observed during failed caboose fetches for a block",
 		Buckets: durationPerBlockHistogram,
 	})
 
+	fetchDurationCarFailureMetric = prometheus.NewHistogram(prometheus.HistogramOpts{
+		Name:    prometheus.BuildFQName("ipfs", "caboose", "fetch_duration_car_failure"),
```

Comment:
We need `fetch_duration_car_peer_failure` version as well (to match what we already have for blocks)